bin/varnishd/cache/cache_http.c

```diff
@@ -821,36 +821,158 @@ http_GetHdrField(const struct http *hp, hdr_t hdr,
 
 /*--------------------------------------------------------------------*/
 
+static ssize_t
+http_parse_uint(const char *b, const char **e)
+{
```

Comment:
This looks like something which belongs in VNUM ?

Reply:
It does, but this patch dates back to February and then I hadn't considered reuse beyond `http_GetContentStuff()`.

On a similar note: https://github.com/varnishcache/varnish-cache/issues/3645#issuecomment-880538864